src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final Random RANDOM = new Random();
+    public static final int MAX_VALUE = 100;
```

Comment:
they can be private as they not used outside of this class
```suggestion
    private static final Random RANDOM = new Random();
    private static final int MAX_VALUE = 100;
```